firmware/common/portapack_persistent_memory.cpp

```diff
@@ -400,6 +404,7 @@ void defaults() {
     set_config_disable_external_tcxo(false);
     set_encoder_dial_sensitivity(DIAL_SENSITIVITY_NORMAL);
     set_config_speaker_disable(true);  // Disable AK4951 speaker by default (in case of OpenSourceSDRLab H2)
+    set_fake_brightness_level(BRIGHTNESS_100);
```

Comment:
It's not necessary to do a "set" here if it's setting the value to 0.  See comment a few lines above.  I don't think it really matters what we put here anyway since the "apply_fake_brightness" bit will default to Off.

Reply:
Thanks! fixed